code/iLasso_for_SCINGE.m

```diff
@@ -79,6 +79,10 @@
 for_metrics.w = w;
 for_metrics.a0 = fit.a0;
 for_metrics.bic = BIC;
-for i = 1:P
-    result(i, :) = w((i-1)*L0+1:i*L0);
+if isempty(w)
```

Comment:
Do we want to print a warning or error message if this case is triggered?  Setting the results to the zeros matrix is a good solution.

Reply:
glmnet provides its own warning message when such an event occurs.